xarray/coding/strings.py

```diff
@@ -249,15 +250,15 @@ def shape(self) -> tuple[int, ...]:
     def __repr__(self):
         return f"{type(self).__name__}({self.array!r})"
 
-    def _vindex_get(self, key):
+    def _vindex_get(self, key: tuple[Any, ...]):
```

Comment:
A lot of `tuple[Any, ...]`! What do you think of using variable name instead, `_IndexerKey = tuple[Any, ...]`? The intention is they should all have the same typing, right?

I worry when it starts repeating this much there's a risk the types starts diverging in future PRs.